### PR TITLE
Replace all occurences of the Greek lower case letter 'mu' with the c…

### PR DESCRIPTION
**Pull request by JenTechSystems**
…orrect unicode Micro symbol.

Add tests to ensure all units that use the Micro scale are using the correct unicode character.

### CODE REVIEW

**Comment by JenTechSystems** [comment]
What exactly is this build message telling me? 
Is it expecting a new unit test class in the UnitsNet.Tests project?

File: UnitsNet.Tests/UnitSystemTests.cs [hunk 1]
```diff
@@ -257,6 +257,64 @@ public string ScientificNotationUpperIntervalFormatting(double value)
         }
 
         [Test]
+        public void ShouldUseCorrectMicroSign()
+        {
```
Comment (anjdreas) [inline UnitsNet.Tests/UnitSystemTests.cs]: Nice set of tests this, even though it will get out of date at some point it is a nice sanity check.

**Comment by anjdreas** [comment]
Just a notice that I have a severe case of the manflu and might need some extra time to respond the next few days. As for the message, my reverse proxy to make the TeamCity build server available on the Internet broke, so you won't be able to access it.

File: UnitsNet.Tests/UnitSystemTests.cs (hunk 1 above)
Comment (JenTechSystems) [inline UnitsNet.Tests/UnitSystemTests.cs]: Yea, it would have been ideal to tap into the code generation and have this test generated for each unit that uses the Micro symbol. The amount I have to contribute to this is dictated by the return on investment for my client though...so that extra level didn't quite make the cut (yet) unfortunately.

**Comment by anjdreas** [comment]
I found the problem, the UnitsNetException type is internal in WINDOWS_UWP profile (Windows Runtime Component target). I added an assembly attribute to make it visible to tests.

**Comment by anjdreas** [comment]
For your reference: d2f0c0825ed2bae4270e6b094f9946d357eda07c
